R/admiral_options.R

```diff
@@ -0,0 +1,194 @@
+admiral_default <- new.env(parent = emptyenv())
+admiral_default$admiral_options <- list(
+  subject_keys = vars(STUDYID, USUBJID)
+  # future_input = vars(...) nolint
+
+  ######################################################
+  ### To enhance feature and add inputs as necessary ###
+  ######################################################
+
+  # 1. Add additional options such as future_input as shown commented above
+  # 2. Update @params with future_input in set_admiral_options roxygen documentation
+  # 3. Modify the commented sections of set_admiral_options()
+)
+
+#' Call a package-standard input
+#'
+#' Call a package-standard input that can be modified for advanced users.
+#'
+#' @param input An unquoted expression of commonly used admiral function inputs.
+#'
+#'   As of now, support only available for `subject_keys`.
+#'
+#' @details
+#' This function allows flexibility for function inputs that may need to be repeated
+#' multiple times in a script, such as for `subject_keys`.
+#'
+#' @author Zelos Zhu
+#'
+#' @return
+#' Currently `subject_keys` returns `vars(STUDYID, USUBJID)` unless user changes the default
+#' using `set_admiral_options()`.
```

Comment:
I would phrase it more general to avoid that we have to update it when we add another admiral option. For example, "The value of the specified option"